cli/cmd/cluster/certificate-rotator.go

```diff
@@ -0,0 +1,229 @@
+package cluster
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/kinvolk/lokomotive/pkg/k8sutil"
+	"github.com/kinvolk/lokomotive/pkg/platform"
+)
+
+type certificateRotatorConfig struct {
```

Comment:
Why this can't be just a single struct? We embed one into the other, both are unexported and we do the validation `certificateRotator` anyway.